### ISSUE TITLE
enable/disable not disabling rotatable

### ISSUE THREAD

**Issue by JD-Robbs** [godswearhats/jquery-ui-rotatable#39]
Hi there,

Nifty little plugin!

I noticed that when using `rotatable({disabled:true})` and `rotatable('enable')`, the plugin adds and removes the correct CSS-class state (`.ui-rotatable-disabled`).

That said, the respective element is, in fact, still rotatable despite the following code:

```
rotateElement: function(event) {
    if (!this.element || this.element.disabled) {
        return false;
    }
    [...]
```

Not sure why it isn't working, but maybe the disabled-flag is stored in an `options` object attached to the DOM element rather than the `disabled` property directly on it?

**Comment by RedShift1** [comment]
Yep, seeing the same here

**Comment by godswearhats** [comment]
Thanks for reporting. I'll see if I can track down the bug in time for the 1.1 release on March 1st.

**Comment by mkdizajn** [comment]
Hi, I'm not sure if this is related to my problem, but I'm trying to make custom implementation for knob to rotate for 270 deg, and then to stop but I think that's not working as expected. The code that trubles me is in JS section on lines 82 - 85. Thanks for great plugin btw :)

http://codepen.io/mkdizajn/pen/KVYNbb?editors=1010

**Comment by godswearhats** [comment]
Fixed in PR #61
